src/module/party/__tests__/sheet-party.test.ts

```diff
@@ -0,0 +1,236 @@
+/**
+ * @file Contains tests for Party Sheet.
+ */
+// eslint-disable-next-line prettier/prettier, import/no-cycle
+import { QuenchMethods } from "../../../e2e";
+import {
+  createMockActorKey,
+  openWindows,
+  waitForInput,
+} from "../../../e2e/testUtils";
+import OsePartySheet from "../party-sheet";
+
+export const key = "ose.sheet.party";
+export const options = { displayName: "Sheet: Party" };
+
+const createMockActor = async (type: string, data: object = {}) =>
+  createMockActorKey(type, data, key);
+
+export default ({ describe, it, expect, assert, after }: QuenchMethods) => {
+  describe("defaultOptions()", () => {
+    it("Has correctly set defaultOptions", () => {
+      const partySheet = new OsePartySheet();
+      expect(partySheet.options.classes).contain("ose");
+      expect(partySheet.options.classes).contain("dialog");
+      expect(partySheet.options.classes).contain("party-sheet");
+      expect(partySheet.options.template).contain(
+        "/templates/apps/party-sheet.html"
+      );
+      expect(partySheet.options.width).equal(280);
+      expect(partySheet.options.height).equal(400);
+      assert(partySheet.options.resizable);
+      expect(partySheet.options.dragDrop[0].dragSelector).equal(
+        ".actor-list .actor"
+      );
+      expect(partySheet.options.dragDrop[0].dropSelector).equal(
+        ".party-members"
+      );
+      assert(!partySheet.options.closeOnSubmit);
+    });
+  });
+  describe("init()", () => {});
```

Comment:
I'll just mark a few that might have been skipped, because I don't know if it was on purpose, or accidentally

Reply:
Updated with todo